client/keys/export.go

```diff
@@ -25,13 +29,31 @@ func ExportKeyCommand() *cobra.Command {
 				return err
 			}
 
-			armored, err := clientCtx.Keyring.ExportPrivKeyArmor(args[0], encryptPassword)
-			if err != nil {
-				return err
+			hex, _ := cmd.Flags().GetBool(flagHex)
+
+			if !hex {
+				armored, err := clientCtx.Keyring.ExportPrivKeyArmor(args[0], encryptPassword)
+				if err != nil {
+					return err
+				}
+
+				cmd.Println(armored)
+				return nil
+			} else {
+				hexPrivKey, err := clientCtx.Keyring.ExportPrivKeyHex(args[0])
+
+				if err != nil {
+					return err
+				}
+
+				cmd.Println(hexPrivKey)
+				return nil
 			}
 
-			cmd.Println(armored)
-			return nil
 		},
 	}
+
+	cmd.Flags().BoolP(flagHex, "x", false, "Export unarmored hex privkey")
```

Comment:
I can't use -h because that's already reserved for --help.

Is -x okay or is there something else I should use?

Reply:
We should also update the description saying that it's not recommended to export unarmored private keys.

Reply:
Updated to add warning.